net/eoip/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2006-2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eoip
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+PKG_MAINTAINER:=bogdik <bogdikxxx@mail.ru>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/bogdik/openwrt-linux-eoip/releases/download/0.5/
+PKG_HASH:=22f6f3864665adef26c7fbd57543a396108ba2dff1282af8143f18bc2a9912f8
+
+PKG_INSTALL:=1
```

Comment:
```suggestion
PKG_NAME:=eoip
PKG_VERSION:=0.5
PKG_RELEASE:=1

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/bogdik/openwrt-linux-eoip/releases/download/0.5/
PKG_HASH:=22f6f3864665adef26c7fbd57543a396108ba2dff1282af8143f18bc2a9912f8

PKG_MAINTAINER:=bogdik <bogdikxxx@mail.ru>
PKG_LICENSE:=GPL-3.0-or-later
PKG_INSTALL:=1
```

Reply:
As mentioned above, you should use in `PKG_MAINTAINER` your real name.

Reply:
@hnyman This was not applied. :(

Reply:
> This was not applied. :(

What was not applied?

`PKG_MAINTAINER:=Bogdan Shatik <bogdikxxx@mail.ru>`

Reply:
Yes, he corrected the maintainer, but he didn't apply my suggestion to increase the stylistic of Makefile but never mind. And would be great if he can send his changes to upstream repository, so we know that the upstream repository is not dead.